src/components/Counter.tsx

```
@@ -0,0 +1,33 @@
+  const increment = () => setCount(count + step);
+  const decrement = () => setCount(count - step);
+  const reset = () => setCount(initialValue);
+
+  return (
+    <div className="counter">
+      <h2>{label}</h2>
+      <div className="counter-display" data-testid="counter-value">
+        {count}
+      </div>
+      <div className="counter-controls">
+        <button onClick={decrement} aria-label="Decrement">-</button>
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)